editor/project_converter_3_to_4.cpp

```diff
@@ -2860,9 +2859,9 @@ String ProjectConverter3To4::get_starting_space(const String &line) const {
 	return empty_space;
 }
 
-// Return object which execute specific function
-// e.g. in `var roman = kieliszek.funkcja()` to this function is passed everything before function which we want to check
-// so it is `var roman = kieliszek.` and this function return `kieliszek.`
+// Return object executing the function in the line.
+// e.g. in `var roman = kieliszek.funkcja()` to this function is passed everything before function which we want to check,
+// so it is `var roman = kieliszek.` and this function returns `kieliszek.`
```

Comment:
I would replace those names with English ones. Additionally, I would simplify the explanation and replace the grave accents with double quotes:

```cpp
// e.g. If you passed "var foo = bar.baz()" to this function, it would return "bar".
```

Reply:
Keeping the original language of the author for this one. as a comment it's not that big of a deal and is kind of charming.